Packs/Kafka/Integrations/Kafka_V2/Kafka_V2.py

```diff
@@ -178,11 +178,15 @@ def print_topics(client):
         for topic in kafka_topics:
             partitions = []
             for partition in topic.partitions.values():
-                partitions.append({
-                    'ID': partition.id,
-                    'EarliestOffset': partition.earliest_available_offset(),
-                    'OldestOffset': partition.latest_available_offset()
-                })
+                partition_output = {'ID': partition.id}
+                try:
```

Comment:
Maybe we should have a param if to perform this logic. It looks like for a system with a large number of topics that can take time.